setup/index.js

```diff
@@ -34,6 +34,18 @@ async function setup() {
     console.log('\n Exiting setup...');
   }
 
+  const { shouldInitializeGitRepository } = await prompt({
+    type: 'confirm',
+    name: 'shouldInitializeGitRepository',
+    message: 'Would you like to initialize this project as a git repository?',
+  });
+
+  const { shouldInitializeCocoapods } = await prompt({
+    type: 'confirm',
+    name: 'shouldInitializeCocoapods',
+    message: 'Would you like to use cocoapods in the ios app?',
+  });
```

Comment:
we can take this out, as all new apps will use cocoapods

Reply:
Good call, removed!